flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -1193,7 +1195,12 @@
 	
 	
 	// ------------------------- JobManager Web Frontend ----------------------
-	
+
+	/** The config key for the address of the JobManager web frontend. */
+	public static final ConfigOption<String> DEFAULT_JOB_MANAGER_WEB_FRONTEND_ADDRESS =
```

Comment:
Just saw that we already have a `WebMonitorConfig` class, but it is not used for all the options, which is too bad. We should gather all web frontend options there (this would be a follow up).